.eslintrc.json

```diff
@@ -1,9 +1,34 @@
 {
+  "root": true,
   "env": {
     "browser": true,
     "es6": true,
     "node": true
   },
-  "extends": ["plugin:prettier/recommended"],
-  "parser": "babel-eslint"
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "extends": ["eslint:recommended", "plugin:jest/recommended", "prettier"],
+  "plugins": ["jest"],
+  "overrides": [
+    {
+      "files": ["**/*.ts"],
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:jest/recommended",
+        "prettier",
+        "prettier/@typescript-eslint"
+      ],
+      "parser": "@typescript-eslint/parser",
+      "plugins": ["jest", "@typescript-eslint"],
+      "rules": {
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/explicit-module-boundary-types": "off",
+        "jest/no-try-expect": "off",
+        "jest/no-conditional-expect": "off"
+      }
+    }
+  ]
```

Comment:
We don't need this.

Reply:
It’s pretty useful to validate the typings. But we don’t need it, of course. It’s very hard to find a “good practice” way to add types. Everybody seems to do it differently.

Reply:
You're right. Let's keep them though.